src/main/java/marquez/api/GetJobRunRequest.java

```diff
@@ -0,0 +1,19 @@
+package marquez.api;
```

Comment:
TODO(code review): Not necessary b/c we already get it from the path (there is no body)